Src/Test.UnitTests.Sarif.PatternMatcher/PatternInvariantTests.cs

```diff
@@ -60,42 +63,90 @@ public static void VerifyAllValidatorsExist(string definitionsFilePath)
                         string.Join($",{Environment.NewLine}  ", rulesWithoutValidators));
         }
 
-        public static void VerifyAllRuleFilenamesMatchDefinitions(string definitionsFilePath)
+        /// <summary>
+        /// This function ensures that for each shared strings variable definition, 
+        /// there is a corresponding regex call to it in the JSON.
+        /// </summary>
+        public static void VerifyAllJsonRulesExist(string definitionsFilePath, string sharedStringsFilePath)
+        {
+            // This function verifies that for each shared string variable definition, there is a rule in the JSON that uses it
+            string definitionsFileContents = File.ReadAllText(definitionsFilePath);
```

Comment:
Repeat of the summary above. This can be removed.

Reply:
I'm not seeing an update yet. If you have it fixed locally, please marked as `pending` until you've pushed to track.